[2.2] docs: Clarify metallb config can be deployed to attached clusters (#4100)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/networking/load-balancing/index.md

MetalLB can be configured in two modes: Layer2 and BGP.

First, set the `WORKSPACE_NAMESPACE` environment variable using the following command to get the name of the workspace's namespace that you would like to deploy MetalLB to.

> MetalLB will be deployed to all attached clusters within this workspace.

Reviewer: ```suggestion
MetalLB is deployed to all attached clusters within this workspace.
```
